build.sbt

```diff
@@ -4495,10 +4495,12 @@ lazy val `runtime-version-manager` = project
     libraryDependencies ++= Seq(
       "com.typesafe.scala-logging" %% "scala-logging"    % scalaLoggingVersion,
       "org.apache.commons"          % "commons-compress" % commonsCompressVersion,
+      "org.apache.tika"             % "tika-core"        % tikaVersion,
```

Comment:
why is this part of the changeset?

Reply:
Shouldn't be. I think it's a bad merge or something. Will fix, thank you!